### ISSUE TITLE
Migrate OIG page information from old page over to new child pages coming off the OIG new template 

### ISSUE THREAD

**Issue by kathycarothers** [fecgov/fec-cms#4055]
### Summary

**What we're after:**
As part of the new design for the OIG page, we will need to migrate information off their page to create new child pages to house the information:

Topics that will need new child pages of the landing page:

- Office of the Inspector General Mission
- Core Values and Organization Chart 
- Audits, Inspections, Special Reviews
- Investigations 
- Whistleblower Protections
- OIG Authority 
- Search OIG Reports
- How to submit a complaint 

### Related issues

- #Ticket number for the template needs to be inputted here 

### Completion criteria

- [ ] To create child pages for each of the topics listed above off the new OIG landing page. These new child pages will represent each of the 6 cards on the landing page. The template for the child pages should include a video embedding feature, for future videos to be included on the pages. The wording will come from https://www.fec.gov/office-inspector-general/
- [ ] Note for the How to submit a complaint page the information will come from the part of the Investigations sub-topic of the old page. 
- [ ] Submit pages for review to the OIG 
- [ ] Change the slugs for the old page and new page in order for the blue breadcrumb to appear (See Pat's email dated 1/11/21 which will require a Wagtail redirect to be put in place. 
- [ ] Publish pages once all are in agreement 

### Future work

_Publish the new landing page and draft redirect ticket for the new page _

**Comment by patphongs** [comment]
Waiting on final code for this page to be merged in before we can start building and migration

**Comment by JonellaCulmer** [comment]
This is done. Closing.
